docs-conceptual/azureadps-1.0/overview.md

```diff
@@ -16,13 +16,11 @@ ms.custom: posh-docs-conceptual
 # Azure Active Directory (MSOnline)
 
 >[!IMPORTANT]
-> MSOnline is planned for deprecation. For more details on the deprecation plans, see the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456). You can start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview) to interact with Azure AD as you would in MSOnline. In addition, Microsoft Graph PowerShell allows you access to all Microsoft Graph APIs and is available on PowerShell 7. For answers to frequent migration queries, see the [migration FAQ](../azureadps-2.0/migration-faq.yml).
+> MSOnline is planned for deprecation on **March 30, 2024**. For more details on the deprecation plans, see the [deprecation update](https://techcommunity.microsoft.com/t5/microsoft-entra-azure-ad-blog/microsoft-entra-new-feature-and-change-announcements/ba-p/3796396). We encourage you to continue migrating to [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview), which is the recommended module for interacting with Azure AD. In addition, Microsoft Graph PowerShell allows you access to all Microsoft Graph APIs and is available on PowerShell 7. For answers to frequent migration queries, see the [migration FAQ](migration-faq.yml).
 
 You can use the Azure Active Directory module for Windows PowerShell cmdlets for Azure AD administrative tasks such as user management, domain management and for configuring single sign-on.
 This topic includes information about how to install these cmdlets for use with your directory.
 
-Please note that we will begin to deprecate this module when the functionality of this module is migrated to the [Azure Active Directory PowerShell for Graph](/powershell/azure/active-directory/install-adv2?view=azureadps-2.0&preserve-view=true). We advise customers who are creating new PowerShell scripts to use the newer module instead of this module.
-
 ## Install the Azure AD Module
 
```

Comment:
This is confusing since we don't want to encourage customers to start installing the Azure AD module, yet deprecation in ongoing.